src/components/TryExtensionLinkDialog.tsx

```diff
@@ -100,11 +100,30 @@ export const TryExtensionLinkDialog: FunctionComponent<{
 const TryExtensionBox: FunctionComponent<{
   onInstallClick: () => void
 }> = props => {
+  const [isClicked, setIsClicked] = useState(false)
+
+  const clickHandler = () => {
+    if (isClicked) {
+      window.location.reload()
+    } else {
+      props.onInstallClick()
+      setIsClicked(true)
+    }
+  }
+
   return (
     <div class="-cbwsdk-extension-dialog-box-top">
       <div class="-cbwsdk-extension-dialog-box-top-install-region">
         <h2>Try the Coinbase Wallet extension</h2>
-        <button onClick={props.onInstallClick}>Install</button>
+        {isClicked && (
+          <div class="-cbwsdk-extension-dialog-box-top-subtext">
+            After installing Coinbase Wallet, refresh the page and connect
+            again.
```

Comment:
nit:
```suggestion
            After installing Coinbase Wallet, refresh the page and connect again.
```

Reply:
![image](https://user-images.githubusercontent.com/50590950/159040977-12237a73-efca-4d5e-81c8-7cbe62635ec5.png)
 
I think this is due to our prettier setting, `printWidth` is set at 80